mix.exs

```diff
@@ -10,7 +10,7 @@ defmodule VegaLite.MixProject do
       version: @version,
       description: @description,
       name: "VegaLite",
-      elixir: "~> 1.7",
+      elixir: "~> 1.12",
```

Comment:
I think it's fine to bump to like 1.10, but `vega_lite` itself could be used in a Phoenix app (outside livebook), so we shouldn't be aggressive with the version with the version. Basically, let's bump to get whichever feature we want to have and not more :)

Reply:
I want to have the `zip_with`! But I can refactor if it's better!

Reply:
I think v1.12 is fine since v1.10 is no longer supported and v1.11 is the minimum supported version at the moment. So one after is ok. :)